lib/Sema/CSSimplify.cpp

```diff
@@ -3425,8 +3426,9 @@ performMemberLookup(ConstraintKind constraintKind, DeclName memberName,
     // See if we have an instance method, instance member or static method,
     // and check if it can be accessed on our base type.
     if (decl->isInstanceMember()) {
-      if ((isa<FuncDecl>(decl) && !hasInstanceMethods) ||
-          (!isa<FuncDecl>(decl) && !hasInstanceMembers)) {
+      if (((isa<FuncDecl>(decl) && !hasInstanceMethods)
+           || (!isa<FuncDecl>(decl) && !hasInstanceMembers))
```

Comment:
Nit: this is not how clang-format indents operators - they are always left at the end of the line.

Reply:
Let me run it on all the files I have changed again!